tests/test_intl.py

```diff
@@ -621,8 +622,11 @@ def test_html_meta(app):
     assert expected_expr in result
     expected_expr = '<meta content="I18N, SPHINX, MARKUP" name="keywords" />'
     assert expected_expr in result
-    expected_expr = '<p class="caption"><span class="caption-text">HIDDEN TOC</span></p>'
-    assert expected_expr in result
+    if docutils.__version_info__ < (0, 17):
+        expected_expr = '<p class="caption"><span class="caption-text">HIDDEN TOC</span></p>'
+        assert expected_expr in result
+    else:
+        expected_expr = '<p><span class="caption-text">HIDDEN TOC</span></p>'
```

Comment:
As a side-effect of migration to `<figcaption>`, the caption of toctree is also rendered using `<figcaption>`, and it's broken (closing tag is missing). It would be better to use another node for toctree caption.